src/aliceVision/matching/ArrayMatcher_hnswlib.hpp

```diff
@@ -0,0 +1,167 @@
+// This file is part of the AliceVision project.
```

Comment:
Please keep the original copyright header:
https://github.com/openMVG/openMVG/blob/cf5a5754926b7e73c9e1d0f67fac296db2365e5f/src/openMVG/matching/matcher_hnsw.hpp#L1-L7. You didn't change the copyright header of HNSNlib, right?. IT is not a a fair open-source citizenship...

Reply:
Please keep a welcoming environment. This is a PR from students who are making their _first open source contribution_ without any bad intention.

For the header, as you and Pierre have been the original authors, then it has been integrated and maintained in OpenMVG, then integrated and maintained in AliceVision, I would suggest:
```
// This file is part of the AliceVision project.
// Copyright (c) 2020 AliceVision contributors.
// Copyright (c) 2019 openMVG contributors.
// Copyright (c) 2019 Romain Janvier and Pierre Moulon
// This Source Code Form is subject to the terms of the Mozilla Public License,
// v. 2.0. If a copy of the MPL was not distributed with this file,
// You can obtain one at https://mozilla.org/MPL/2.0/.
```
@pmoulon @rjanvier Is that fine to you?

Reply:
fine, thanks.

Reply:
Apologies for this, we did not notice it; thank you for pointing it @rjanvier :). We will add the copyright header @fabiencastan suggested above.